src/tools/singlejar/zip64_test.sh

```diff
@@ -53,7 +53,15 @@ else
 fi
 
 singlejar="$(rlocation "io_bazel/src/tools/singlejar/singlejar${EXE_EXT}")"
-jartool="$(rlocation "local_jdk/bin/jar${EXE_EXT}")"
+javabase="$1"
+if [[ $javabase = /* ]]; then
```

Comment:
Code assumes `$1` is Unix-style path. What about Windows paths?

Reply:
I am happy to add a code snippet for Windows to this change if you tell me what I should add :-). I don't think it's a necessary part of this PR though.

Reply:
if [[ $javabase = /* || $javabase =~ [A-Za-z]:[/\\] ]]; then

Reply:
thanks! done